docs/getting-started-auditor.md

```diff
@@ -0,0 +1,170 @@
+Please check [the configuration file](https://github.com/scalar-labs/scalar/blob/master/auditor/conf/auditor.properties.tmpl) for more detail.
+
+
+## Start Ledger and Auditor
+
+Please start Ledger and Auditor in your own way.
+For example, if you use the built-in command line tools, do as follows:
+
+Ledger:
+```
+bin/scalar-ledger --properties ledger.properties
+```
+
+Auditor:
+```
+bin/scalar-auditor --properties auditor.properties
+```
+
+You can also use the containers that are available in [the GitHub Container Registry](https://github.com/orgs/scalar-labs/packages).
+See [saclardl-samples](https://github.com/scalar-labs/scalardl-samples) repo for more details about how to use the containers.
+
+## Register each certificates of Ledger and Auditor
+
+As we explained,
+Ledger needs to register its certificate to Auditor, and Auditor needs to register its certificate to Ledger.
+This can be done by registering it as a client as follows:
+
+Ledger registers its certificate to Auditor
+```
+bin/register-cert --properties client.properties.ledger
+```
+
+Auditor registers its certificate to Ledger
+```
+bin/register-cert --properties client.properties.auditor
+```
+
+Please configure `client.properties.ledger` and `client.properties.auditor` properly.
+(This document assumes you know how to configure client.properties properly.)
+Note that `scalar.dl.client.cert_holder_id` has to be properly set to match with the one configured in Ledger and Auditor respectively.
+
+## Register your certificate
+
+You also need to update the properties file of a client before interacting with Scalar DL with Auditor.
+Please update the following entries:
+```
+[client.properties]
+scalar.dl.client.auditor.enabled=true
+scalar.dl.client.auditor.host=localhost
+```
+
+Then, you can register your certificate just like you have been doing usually (without Auditor).
+
+```
+client/bin/register-cert --properties client.properties
+```
+
+Note that this registers the certificate to both Ledger and Auditor.
+
+## Register your contracts
+
+For registering contracts, you can also do as usual.
+
+```
+client/bin/register-contract --properties client.properties --contract-id StateUpdater --contract-binary-name com.org1.contract.StateUpdater --contract-class-file build/classes/java/main/com/org1/contract/StateUpdater.class
+```
+
+Note that this registers the contract to both Ledger and Auditor.
+
+## Execute the contract
+
+Now you are ready to execute the contract with the following command as usual.
+
+```
+client/bin/execute-contract --properties client.properties --contract-id StateUpdater --contract-argument '{"asset_id":"some_asset", "state":3}'
+```
+
+Note that this triggers a little complex protocol between Ledger and Auditor to make them go to the same states without trusting each other.
+During the execution, it may detect inconsistencies between them if there is a tampering in one of the components.
```

Comment:
Can just be "there is tampering in one of the components" here.

Reply:
Probably fixed in fea2a17ffdea2b1f6b1224c7141b3d29f3fbeffa.